flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/plan/OperationTreeBuilder.scala

```diff
@@ -369,6 +369,14 @@ class OperationTreeBuilder(private val tableEnv: TableEnvironment) {
     Union(left.asInstanceOf[LogicalNode], right.asInstanceOf[LogicalNode], all).validate(tableEnv)
   }
 
+  def map(mapFunction: Expression, child: TableOperation): TableOperation = {
+    val childNode = child.asInstanceOf[LogicalNode]
+    val expandedFields = expandProjectList(
+      Seq(mapFunction).map(expressionBridge.bridge)
+        .map(Flattening), childNode, tableEnv)
```

Comment:
`.map(expressionBridge.bridge) .map(Flattening)` 
to
 `.map(e => Flattening(expressionBridge.bridge(e)))`
I am not pretty which one is better? just a suggestion. :)
What do you think?